src/phoenix/experimental/evals/functions/classify.py

```diff
@@ -64,6 +64,8 @@ def llm_classify(
     verbose: bool = False,
     use_function_calling_if_available: bool = True,
     provide_explanation: bool = False,
+    include_prompt: bool = False,
+    include_response: bool = False,
```

Comment:
thought for the future: If we end up wanting to curate the output of the dataframe it might be simpler to just have output_columns. This works for now though